app/controllers/users_controller.rb

```diff
@@ -76,12 +59,12 @@ def destroy
 
   def impersonate
     impersonate_user(@user)
-    redirect_to users_path, notice: "Now impersonating #{@user.name}."
+    redirect_to user_path(@user), notice: "Now impersonating #{@user.name}."
```

Comment:
I am curious about this change in behaviour. The impersonating buttons are still in the index list, right? What is the reasoning for redirecting to the user profile now?

Reply:
We could use the community_index_path as well. I thought that would be more intuitive the user be redirected to the page of the user he chooses to impersonate, but I can change it.

Reply:
Yes please! 👍  I'd appreciate that. 💃

Reply:
@carpodaster about the redirect I asked you. Do you prefer to keep the old behavior?

Reply:
@branquinhoaa I don't mind, really. I think it makes sense to redirect to the person whose identity you've just assumed but it's also changing a behaviour whereas everything else in this PR basically maintains the old behaviour, just moves it between controllers.

@F3PiX seems to be in favour of keeping the existing behaviour, so let's roll with that.